config/config.go

```diff
@@ -99,6 +99,7 @@ type ICMPProbe struct {
 	PreferredIPProtocol string `yaml:"preferred_ip_protocol,omitempty"` // Defaults to "ip6".
 	PayloadSize         int    `yaml:"payload_size,omitempty"`
 	DontFragment        bool   `yaml:"dont_fragment,omitempty"`
+	SourceAddress       string `yaml:"source_address,omitempty"`
```

Comment:
This should apply to all probes

Reply:
@brian-brazil Okay... Added similar code for tcp, http, dns and moved the config to the module.